web/src/lib/components/StallView.svelte

```diff
@@ -87,27 +147,62 @@
                 {/if}
             </div>
         </div>
+        <!-- if stallOwner viewing own stall -->
+        {#if $user && $user.nym == stallOwner.nym}
+        <div>
+            <div class="pt-10 flex justify-center items-center">
+                <section class="w-11/12 lg:w-3/5">
+                    {#if currentEntity}
+                        <svelte:component this={editor} bind:entity={currentEntity} onSave={saveCurrentEntity} onCancel={() => currentEntity = undefined} />
+                    {:else if entities === null}
+                        <Loading />
+                    {:else}
+                        {#if showNewButton}
+                            <div class="flex items-center justify-center mb-4">
+                                <div class="glowbutton glowbutton-new" on:click|preventDefault={() => currentEntity = newEntity()}></div>
+                            </div>
+                        {/if}
+                    {/if}
+                </section>
+            </div>
+        </div>
+        {/if}
         {#if entities !== null && entities.length > 0}
         <!-- stallOwner listing section -->
         <hr class="border-solid border-accent divide-y-0 opacity-50 my-5">
         <div class="tabs flex items-center justify-center">
-            {#each ['ACTIVE AUCTIONS', 'PAST AUCTIONS'] as tab}
+            {#each tabList as tab}
                 <li class="tab text-xs font-semibold" class:tab-active={tab === currentTab} class:tab-bordered={tab === currentTab} on:click={() => currentTab = tab}>
                     <div>{tab}</div>
                 </li>
             {/each}
         </div>
         <div class="pt-6 pb-6">
             <div class="grid md:grid-cols-3 grid-cols-1">
-                    {#each entities as auction}
-                        {#if currentTab === 'ACTIVE AUCTIONS' && !auction.ended}
+                    {#each entities as entity}
+                        {#if $user && $user.nym == stallOwner.nym}
```

Comment:
I think this if is useless, because the UNLAUNCHED tab will only exist for the owner anyway, no?